src/controllers/AdminController.ts

```diff
@@ -165,6 +165,41 @@ export class AdminAPI extends Controller {
     return template;
   }
 
+  /**
+   * Create a templates. An overview of Template usage in Retraced can be found at
+   *
+   * https://boxyhq.com/docs/retraced/advanced/display-templates
+   *
+   * @param auth          Base64 encoded JWT
+   * @param projectId     The project id
+   * @param environmentId The environment id
+   * @param body          The template resource to create
+   */
+    @Post("project/{projectId}/templates")
+    @SuccessResponse("201", "Created")
+    public async createTemplates(
+      @Header("Authorization") auth: string,
+      @Path("projectId") projectId: string,
+      @Query("environment_id") environmentId: string,
+      @Body() body: {templates: TemplateValues[]},
+      @Request() req: express.Request
+    ): Promise<TemplateResponse[]> {
+      await audit(req, "template.create.many", "c", {
+        target: {
+          fields: body,
+        },
+      });
+
+      const templates: TemplateResponse[] = []
+      body.templates.map(async (templateToCreate) => {
```

Comment:
Need to limit the maximum number of templates that can be added in one go to 100.

Reply:
Throw an error in case more than 100 template are passed